packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -351,13 +361,21 @@ export class Autosuggest {
     return `${this.inputId}-${this.suggestions.indexOf(suggestion) + 1}`;
   }
 
+  componentDidRender() {
+    if(this.loadingDelayed) {
+      setTimeout(() => {
```

Comment:
Als deze setTimeout loopt en iemand typt ondertussen verder dan is de loading-delay vermoedelijk korter dan de opgegeven waarde.

Ik vraag me verder af of je dit in de componentDidRender wil doen. Kan die niet beter in de onInput? Ik denk dat je de debounce constructie van r.139 kunt herhalen maar dan voor `showLoading`.